.gitignore

```diff
@@ -1,3 +1,5 @@
 /taskcluster-base-test.conf.json
 node_modules/
 .vagrant/
+.test/
+lib/
```

Comment:
Note: I might have been the root of this insanity...
I suggest we add .gitigore and 
.npmignore if people want to... I see both the pros and cons, and see no value in arguing over it :)